<?xml version="1.0"?>
<launch>
  <param name="franka_arm_description" command="$(find xacro)/xacro '$(find franka_description)/robots/panda_arm.urdf.xacro'"/>
  <param name="urdf_param" value="/franka_arm_description"/> 
  <node name="testIRM" pkg="base_placement_planner" type="testIRM" output="screen" >
    <param name="yaml_path" type="string" value="$(find base_placement_planner)/config/hw_param.yaml"/>
    <param name="ws_path" type="string" value="$(find base_placement_planner)/mapConfig/"/> 
    <param name="file_name" type="string" value="ManipulabilityDataAcc.bin"/> <!-- 저장하거나 로드할 파일 이름 입력!!!!!!! -->
  </node>
</launch>